src/app/pages/dashboard/components/test-centre-journal-card/test-centre-journal-card.scss

```diff
@@ -0,0 +1,54 @@
+.test-centre-journal-button {
```

Comment:
other buttons have 'activated' so you can see them clicked
  --background-activated: var(--gds-dark-green-2);

could we not have this in the 'button' theme scss so we can use blue elsewhere if needed